src/shared/views/CometChatBadge/CometChatBadge.tsx

```diff
@@ -16,15 +16,11 @@ import styles from './styles';
  *
  */
 interface CometChatBadgeProps {
-  count: number;
+  count?: number;
```

Comment:
Your type is either incorrect, or you don't need to set a default `count`. It's one or the other.